assets/js/components/data-block.js

```diff
@@ -101,15 +104,19 @@ class DataBlock extends Component {
 					</div>
 				}
 				<div className="googlesitekit-data-block__change-source-wrapper">
-					<div className={ `
-						googlesitekit-data-block__change
-						googlesitekit-data-block__change--${ 0 <= change ? 'positive' : 'negative' }
-					` }>
+					<div className="googlesitekit-data-block__change">
 						{ '' === change && <Fragment>&nbsp;</Fragment> }
-						{ change && [
-							<span key="arrow" className={ `googlesitekit-data-block__arrow ${ reverseArrowDirection ? 'googlesitekit-data-block__arrow--reverse' : '' }` }><SvgIcon id={ `arrow${ changeType }` } height="9" width="9" /></span>,
-							<span key="values" className="googlesitekit-data-block__value">{ `${ Math.abs( change ) }${ changeDataUnit } ${ period }` }</span>,
-						] }
+						{ change && <Fragment>
+							<div className="googlesitekit-data-block__arrow">
```

Comment:
Just curious, why did you turn this into a `div` (from `span` before) and remove the `key` prop?

Reply:
Ah, this was mostly unintentional as the `.googlesitekit-data-block__arrow` was originally removed and then re-added to preserve a style.  Both elements have `display: inline-block` so I don't think it matters which element is used, although I agree that they should be the same.

The `key` prop is only necessary when rendering an array of elements, but this was changed to use a `<Fragment>` instead so it's unnecessary since children never change order.

I'll change it back to a `span` and remove the `key` on the other element for consistency unless you'd prefer otherwise.

Reply:
SGTM 👍

Reply:
Updated!